packages/sku/context/index.js

```diff
@@ -1,23 +1,20 @@
 import { existsSync } from 'node:fs';
 import { join } from 'node:path';
 import chalk from 'chalk';
-import { register } from 'esbuild-register/dist/node';
```

Comment:
I think both `esbuild` and `esbuild-register` can now be removed as dependencies with this change.